components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/pom.xml

```diff
@@ -130,6 +130,14 @@
             <groupId>org.wso2.carbon.consent.mgt</groupId>
             <artifactId>org.wso2.carbon.consent.mgt.core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.event</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.application.auth.basic</groupId>
```

Comment:
We can't have a dependency for org.wso2.carbon.identity.application.authenticator.basicauth in here as it will result in a cyclic dependency.